src/main/java/core/BrowserService.java

```diff
@@ -0,0 +1,82 @@
+package core;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import io.github.bonigarcia.wdm.config.DriverManagerType;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.firefox.FirefoxOptions;
+import org.openqa.selenium.support.events.EventFiringDecorator;
+import org.openqa.selenium.support.events.WebDriverListener;
+import utils.Listener;
+import utils.Waiter;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.NoSuchElementException;
+
+public class BrowserService {
+    private WebDriver driver;
+
+    public BrowserService(String browserType) {
+        DriverManagerType driverManagerType;
+        if (browserType.equalsIgnoreCase("Chrome")) {
+            driverManagerType = DriverManagerType.CHROME;
+            WebDriverManager.getInstance(driverManagerType).setup();
+            HashMap<String, Object> chromePrefs = new HashMap<String, Object>();
+            chromePrefs.put("profile.default_content_settings.popups", 0);
+            chromePrefs.put("download.default_directory", System.getProperty("user.dir") + File.separator + "src" + File.separator + "test" + File.separator + "resources" + File.separator + "DownloadedFiles");
+            chromePrefs.put("download.prompt_for_download", false);
```

Comment:
all these capabilities should be placed in a separate file (f.e. chromeOptions.properties)
There should not be any hardcoded string here